core/usage_interceptor.go

```diff
@@ -99,18 +100,33 @@ func (t *Textile) preUsageFunc(ctx context.Context, method string) (context.Cont
 	cus, err := t.bc.GetCustomer(ctx, account.Owner().Key)
 	if err != nil {
 		if strings.Contains(err.Error(), mongo.ErrNoDocuments.Error()) {
-			opts := []billing.Option{
-				billing.WithEmail(account.User.Email),
+			email, err := t.getAccountCtxEmail(ctx, account)
+			if err != nil {
+				return ctx, err
```

Comment:
Is it ok to `return ctx, err` instead of maybe safer `return nil, ...` done in other if-branches below?

Reply:
Yes, but it might be more clear to just return `nil, err` everywhere. I'll make a note of that and do it in another change.